server/src/main/java/io/crate/expression/scalar/postgres/PgEncodingToChar.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.postgres;
+
+import io.crate.data.Input;
+import io.crate.expression.scalar.ScalarFunctionModule;
+import io.crate.metadata.FunctionName;
+import io.crate.metadata.NodeContext;
+import io.crate.metadata.Scalar;
+import io.crate.metadata.TransactionContext;
+import io.crate.metadata.functions.Signature;
+import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
+import io.crate.types.DataTypes;
+
+import static io.crate.metadata.functions.Signature.scalar;
+
+public class PgEncodingToChar extends Scalar<String, Integer> {
+    public static final String NAME = "pg_encoding_to_char";
+    private static final FunctionName FQN = new FunctionName(PgCatalogSchemaInfo.NAME, NAME);
+
+    public static void register(ScalarFunctionModule module) {
+        module.register(
+            scalar(
+                FQN,
+                DataTypes.INTEGER.getTypeSignature(),
+                DataTypes.STRING.getTypeSignature()
+            ).withFeatures(NO_FEATURES),
+            PgEncodingToChar::new
+        );
+    }
+
+    private final Signature signature;
+    private final Signature boundSignature;
+
+    public PgEncodingToChar(Signature signature, Signature boundSignature) {
+        this.signature = signature;
+        this.boundSignature = boundSignature;
+    }
+
+    @Override
+    public Signature signature() {
+        return signature;
+    }
+
+    @Override
+    public Signature boundSignature() {
+        return boundSignature;
+    }
+
+    @SafeVarargs
+    @Override
+    public final String evaluate(TransactionContext txnCtx, NodeContext nodeContext, Input<Integer>... args) {
+        return "UTF8";
```

Comment:
It is true that CrateDB currently only supports `UTF-8` as the server-side encoding, this is also hardcoded.
But this function is a conversion function, converting a integer representation of the encoding to a string representation.
Thus we should either support all valid/known/registered integer encoding representations or fail on unsupported ones.